### ISSUE TITLE
Incompatible versions of accesskit

### ISSUE THREAD

**Issue by paholg** [bevyengine/bevy#8258]
## Bevy version

0.10

## \[Optional\] Relevant system information

If you cannot get Bevy to build or run on your machine, please include:

- Rust version: 1.68.1
- OS: Ubuntu 22.04

## What you did

Create a new project, `cargo add bevy`, and try to compile.

## What went wrong

It looks like there are incompatible versions of `accesskit` used by bevy crates; 0.10.1 and 0.11. 0.11 comes from a new release of `accesskit_winit` and, as a workaround, one can downgrade it. This can be done with `cargo update -p accesskit_winit --precise 0.12.4`.

Full error message below.

```
   Compiling bevy_winit v0.10.0
error[E0308]: mismatched types
    --> /home/paho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_winit-0.10.0/src/accessibility.rs:50:17
     |
50   | /                 TreeUpdate {
51   | |                     focus: if event.focused {
52   | |                         Some(focus_id.to_node_id())
53   | |                     } else {
...    |
56   | |                     ..default()
57   | |                 }
     | |_________________^ expected struct `accesskit::TreeUpdate`, found struct `TreeUpdate`
     |
     = note: struct `TreeUpdate` and struct `accesskit::TreeUpdate` have similar names, but are actually distinct types
note: struct `TreeUpdate` is defined in crate `accesskit`
    --> /home/paho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/accesskit-0.10.1/src/lib.rs:2336:1
     |
2336 | pub struct TreeUpdate {
     | ^^^^^^^^^^^^^^^^^^^^^
note: struct `accesskit::TreeUpdate` is defined in crate `accesskit`
    --> /home/paho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/accesskit-0.11.0/src/lib.rs:2336:1
     |
2336 | pub struct TreeUpdate {
     | ^^^^^^^^^^^^^^^^^^^^^
     = note: perhaps two different versions of crate `accesskit` are being used?

error[E0308]: mismatched types
    --> /home/paho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_winit-0.10.0/src/accessibility.rs:147:21
     |
147  | /                     TreeUpdate {
148  | |                         nodes: to_update,
149  | |                         focus: focus_id.map(|v| v.to_node_id()),
150  | |                         ..default()
151  | |                     }
     | |_____________________^ expected struct `accesskit::TreeUpdate`, found struct `TreeUpdate`
     |
     = note: struct `TreeUpdate` and struct `accesskit::TreeUpdate` have similar names, but are actually distinct types
note: struct `TreeUpdate` is defined in crate `accesskit`
    --> /home/paho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/accesskit-0.10.1/src/lib.rs:2336:1
     |
2336 | pub struct TreeUpdate {
     | ^^^^^^^^^^^^^^^^^^^^^
note: struct `accesskit::TreeUpdate` is defined in crate `accesskit`
    --> /home/paho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/accesskit-0.11.0/src/lib.rs:2336:1
     |
2336 | pub struct TreeUpdate {
     | ^^^^^^^^^^^^^^^^^^^^^
     = note: perhaps two different versions of crate `accesskit` are being used?

error[E0277]: the trait bound `WinitActionHandler: accesskit::ActionHandler` is not satisfied
   --> /home/paho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_winit-0.10.0/src/winit_windows.rs:157:13
    |
157 |             Box::new(handler.clone()),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `accesskit::ActionHandler` is not implemented for `WinitActionHandler`
    |
    = note: required for the cast from `WinitActionHandler` to the object type `dyn accesskit::ActionHandler`

error[E0308]: mismatched types
    --> /home/paho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_winit-0.10.0/src/winit_windows.rs:151:17
     |
151  | /                 TreeUpdate {
152  | |                     nodes: vec![(accesskit_window_id, root)],
153  | |                     tree: Some(Tree::new(accesskit_window_id)),
154  | |                     focus: None,
155  | |                 }
     | |_________________^ expected struct `accesskit::TreeUpdate`, found struct `TreeUpdate`
     |
     = note: struct `TreeUpdate` and struct `accesskit::TreeUpdate` have similar names, but are actually distinct types
note: struct `TreeUpdate` is defined in crate `accesskit`
    --> /home/paho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/accesskit-0.10.1/src/lib.rs:2336:1
     |
2336 | pub struct TreeUpdate {
     | ^^^^^^^^^^^^^^^^^^^^^
note: struct `accesskit::TreeUpdate` is defined in crate `accesskit`
    --> /home/paho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/accesskit-0.11.0/src/lib.rs:2336:1
     |
2336 | pub struct TreeUpdate {
     | ^^^^^^^^^^^^^^^^^^^^^
     = note: perhaps two different versions of crate `accesskit` are being used?
```

**Comment by paholg** [comment]
accesskit_winit 0.12.5 has been yanked, solving this issue.
